Adding Labels to Contact Us Minicons

KimberlyMunoz opened the pull request:
Adding labels to the contact us minicons. 

## Additions

- Added labels to the contact us minicons

## Changes

- Moved the phone numbers, email address to their own paragraph instead of a span.
- Moved the address to the same unordered-icon-list format as the rest of the contact information.

## Testing

- /contact-us/

## Review

- @jimmynotjim 
- @anselmbradford 
- @sebworks 

## Preview

![image](https://cloud.githubusercontent.com/assets/1860176/7306380/decf1e2c-e9d2-11e4-922d-89829e2f0dde.png)

![image](https://cloud.githubusercontent.com/assets/1860176/7306399/f9656a02-e9d2-11e4-8324-b62c8f6f06da.png)

## Notes
- My first pull request for cfgov-refresh :facepunch:

jimmynotjim commented:
- Use one label above multi-contact sources (ex: two phone numbers)
- Add label above phone numbers in "Submit a Complaint" section and update vanity phone numbers
- Add tel links to phone numbers

KimberlyMunoz commented:
Fixed the labels for multiple contact sources and added support for tel: links based on what I saw in the footer. Basically the tel: links are there, but only visible on smaller screens.

For the vanity numbers in Submit a Complaint, there's only two fields for phone numbers. I'm going to do a simple replace to add the vanity numbers and so that the end user only needs to update the number and description.

KimberlyMunoz commented:
Content is ramped up. Here's how it looks now and it's ready for a second round of reviews.

![image](https://cloud.githubusercontent.com/assets/1860176/7379981/cbec0980-edc6-11e4-9db6-e2c1a41ee9ec.png)

jimmynotjim commented:
Looking good. One last thing is to add a "Phone:" label with an icon above the "Submit a complaint" numbers to match the "Inquiries" one.

KimberlyMunoz commented:
Thanks for feedback! Added.

KimberlyMunoz commented:
There was an outstanding issue where we discovered the field names for phone numbers and email were wrong. I went ahead and updated them since it makes this PR easier to review.

anselmbradford commented on contact-us/submit-a-complaint.html:
```diff
@@ -7,25 +7,34 @@ <h1 class="h4">
+        {%- if submit_a_complaint.phone_1_num %}
+        <p>
+            <a class="list_link list_link__phone" href="tel:{{ submit_a_complaint.phone_1_num }}">
+                    {{ format_phone(submit_a_complaint.phone_1_num)| replace("2372", "CFPB (2372)") }}
```
Very minor:
This line has an extra indent.

KimberlyMunoz replied:
L25? Fixing!

anselmbradford replied:
L28 https://github.com/cfpb/cfgov-refresh/blob/DF-1803-contact-us-minicons/contact-us/submit-a-complaint.html#L28

KimberlyMunoz replied:
I didn't see that one at all! Thanks for the catch. Fixed!

anselmbradford commented:
This looks good to me :+1: Could you add the changes to the `Unreleased` section in the changelog please? Otherwise, GTG from my end. Might want another check off from @jimmynotjim before squash 'n' merge.

KimberlyMunoz commented:
What do you think, @jimmynotjim?

jimmynotjim commented:
Tested locally and running great. Thanks for including the indexing fix in the contact list as well. :+1:
